examples/elasticsearch/movies_metric.yaml

```diff
@@ -0,0 +1,22 @@
+steps:
+- dataFrameName: RatedMovies
+  sql:
+    SELECT userid,
+           movies.movieid
+    FROM ratings
+    JOIN movies ON ratings.movieid = movies.movieid
+    GROUP BY 1,2
+    LIMIT 10
+- dataFrameName: RatedMoviesJson
+  sql:
+    SELECT to_json(struct(userid, movieid)) AS jsonValue,
+    movieid AS key
+    FROM RatedMovies
+output:
+- dataFrameName: RatedMoviesJson
+  outputType: Elasticsearch
+  outputOptions:
+    saveMode: Append
```

Comment:
Add comment that overwrite is default, maybe it shouldn't be the default?

Reply:
Switched default to Append